config/locales/en/steps.yml

```diff
@@ -165,6 +165,30 @@ en:
       edit:
         page_title: Other Relevant information
         heading: Other Relevant information
+    solicitor_declaration:
+      edit:
+        page_title: Confirm the following
+        heading: Confirm the following
+        subheading_html: |
+          <p class="govuk-body">You agree that:</p>
+          <ul class="govuk-list govuk-list--bullet">
+              <li> you've provided correct and complete information in this claim </li>
+              <li>the work you are claiming for has not been, and will not be, part of any other claim for remuneration from criminal legal aid</li>
+          </ul>
+        warning_html: |
+          <div class="govuk-warning-text">
+            <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+            <strong class="govuk-warning-text__text">
+              <span class="govuk-warning-text__assistive">Warning</span>
+              If you provide wrong or incomplete information, or are found to have committed benefit fraud, you may:
+              <ul class="govuk-list govuk-list--bullet govuk-!-font-weight-bold">
+                <li>be prosecuted</li>
+                <li>need to pay a financial penalty</li>
+              </ul>
+            </strong>
+          </div>
```

Comment:
QQ: Should we be putting HTML into the locales files? Would this not be better in the view with the text here?

Reply:
I would like to put this in the ERB, only reason I've done this way as line 75 'subheading_html' has HTML content and I thought this is the approach we are taking. Happy to change if my understanding is wrong.

Reply:
Saying so, I am changing this as suggested, translation to WELSH would be much simpler.

Reply:
Done!